starport/cmd/network_request_list.go

```diff
@@ -0,0 +1,100 @@
+package starportcmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+)
+
+// NewNetworkRequestList creates a new request list command to list
+// requests for a chain
+func NewNetworkRequestList() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "list [launch-id]",
+		Short: "List all pending requests",
+		RunE:  networkRequestListHandler,
+		Args:  cobra.ExactArgs(1),
+	}
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	c.Flags().AddFlagSet(flagNetworkFrom())
+	c.Flags().AddFlagSet(flagSetHome())
+	return c
+}
+
+func networkRequestListHandler(cmd *cobra.Command, args []string) error {
+	// initialize network common methods
+	nb, err := newNetworkBuilder(cmd)
+	if err != nil {
+		return err
+	}
+	defer nb.Cleanup()
+
+	// parse launch ID
+	launchID, err := strconv.ParseUint(args[0], 10, 64)
+	if err != nil {
+		return errors.Wrap(err, "error parsing launchID")
+	}
+	if launchID == 0 {
+		return errors.New("launch ID must be greater than 0")
+	}
+
+	n, err := nb.Network()
+	if err != nil {
+		return err
+	}
+
+	requests, err := n.FetchRequests(cmd.Context(), launchID)
+	if err != nil {
+		return err
+	}
+
+	// Rendering
+	requestTable := tablewriter.NewWriter(os.Stdout)
```

Comment:
Instead of tablewriter, can we use the new entrywriter? You can either copy the source code of the same package here (faster option) or wait for related PR to be merged: https://github.com/tendermint/starport/pull/1787